builtin/game/misc.lua

```diff
@@ -114,6 +114,10 @@ function core.get_connected_players()
 	return temp_table
 end
 
+function minetest.player_exists(name)
+	return minetest.get_auth_handler().get_auth(name) ~= nil
```

Comment:
if i remember this call is used in some code parts, no ? if yes can you call it ?